Sources/Linq2DynamoDb.DataContext/IEntityWrapper.cs

```diff
@@ -7,5 +7,6 @@ public interface IEntityWrapper
         object Entity { get; }
         Document GetDocumentIfDirty();
         void Commit();
+        Document AsDocument();
```

Comment:
Agreed. We only need the value of the version property (if optimistically locking) instead of the whole document.